### DISCUSSION TITLE
How to solve this issue with the devcontainer?

### DISCUSSION THREAD

**Discussion by Jayvdw60** [community/community#45881] [New to GitHub]
Dear people,

Totally new to Github and I have no idea where to start to solve the issue mentioned on the screenshot. What do I need to do (step by step) to update the devcontainer and to rectify it?
Moreover, why did this configuration error happen? It occurred after an update 

<img width="560" alt="Screenshot 2023-01-31 at 13 46 58" src="https://user-images.githubusercontent.com/121435937/215727787-2152571d-e09a-462f-b9b6-05cbd4a83293.png">

**Comment by jge162** [comment]
hi @Jayvdw60 you need to go and review the logs as warning suggests, update your Devcontainer,json file to fix the config error
then In the Github Codespace, run the `"Rebuild Container"` command. this should fix issue. or if it does not reach out to [GitHub Support](https://support.github.com/request/account?tags=hubberfy_account)

**Comment by MostafaWaleed0** [comment]
The configuration error may have occurred after an update because of changes to the format or structure of the devcontainer.json file that were not properly reflected in your local copy. It is important to always review and update your configuration files after updates to ensure that they continue to work as expected.

```json
{
  "name": "Node.js",
  "image": "mcr.microsoft.com/devcontainers/javascript-node:0-18",
  "features": {
    "ghcr.io/devcontainers/features/github-cli:1": {},
    "ghcr.io/devcontainers-contrib/features/redis-homebrew:1": {}
  },
  "forwardPorts": [
    3001,
    6379
  ],
  "postCreateCommand": "yarn install",
  "postAttachCommand": ".devcontainer/postAttachCommand.sh",
  "customizations": {},
  "remoteUser": "root"
}
```
